app/controllers/wikis_controller.rb

```diff
@@ -28,7 +50,12 @@ def edit
 
   def update
     @wiki = Wiki.find(params[:id])
-    if @wiki.update_attributes(params.require(:wiki).permit(:title, :body))
+    @collaborators = User.where(:id => params[:collaborators])
+    @wiki.users = @collaborators 
+    if @wiki.creator? && !@wiki.users.include?(User.find(@wiki.creator))
```

Comment:
Do you have to do `User.find(@wiki.creator)` or could you just do `@wiki.creator`?